subsys/mpsl/mpsl_init.c

```diff
@@ -171,6 +172,11 @@ static int mpsl_lib_init(const struct device *dev)
 		return err;
 	}
 
+#if CONFIG_MPSL_FEM_DEVICE_CONFIG_254
+	mpsl_fem_device_config_254_apply_set(
+		IS_ENABLED(CONFIG_MPSL_FEM_DEVICE_CONFIG_254));
+#endif
```

Comment:
Shouldn't have the outer #ifs then. IS_ENABLED handles it.
```suggestion
	mpsl_fem_device_config_254_apply_set(
		IS_ENABLED(CONFIG_MPSL_FEM_DEVICE_CONFIG_254));
```

Reply:
Fixed